interfaces/builtin/network_manager.go

```diff
@@ -92,12 +92,9 @@ network packet,
 /etc/netplan/{,**} rw,
 
 # Allow access to configuration files generated on the fly
-# from netplan and let NetworkManager store its DHCP leases
-# in the dhcp subdirectory so that console-conf can access
-# it.
-/run/NetworkManager/ w,
-/run/NetworkManager/{,**} r,
-/run/NetworkManager/dhcp/{,**} w,
+# from netplan and let NetworkManager store its configuration
+# in the same place.
+/run/NetworkManager/{,**} wr,
```

Comment:
Nitpick: please use 'rw'. That said, can you comment on why we need blanket 'w' for the whole dir? I think it is fine, but would like to at least have it documented in the PR (and really, the commit message) on why it was done.